crates/biome_js_analyze/src/lint/style/use_nodejs_import_protocol.rs

```diff
@@ -57,7 +60,15 @@ impl Rule for UseNodejsImportProtocol {
     type Options = ();
 
     fn run(ctx: &RuleContext<Self>) -> Self::Signals {
-        let module_name = ctx.query().module_name_token()?;
```

Comment:
Is it possible to enhance the `module_name_token` method and let it return `None` for this case?  I see this method is used in several places and this issue does not seem to be specific to this rule.

https://biomejs.dev/playground/?lintRules=all&code=ZABlAGMAbABhAHIAZQAgAG0AbwBkAHUAbABlACAAIgBuAG8AZABlADoAYQBzAHMAZQByAHQAIgAgAHsAfQA%3D

Reply:
Good idea but I'd like to put them into another helper function.